PathfinderAPI/BaseGameFixes/ListingServerFixes.cs

```diff
@@ -0,0 +1,48 @@
+﻿using Hacknet;
+using HarmonyLib;
+using Mono.Cecil.Cil;
+using MonoMod.Cil;
+
+namespace Pathfinder.BaseGameFixes
+{
+    [HarmonyPatch]
+    internal static class ListingServerFixes
+    {
+        [HarmonyPrefix]
+        [HarmonyPatch(typeof(MissionListingServer), nameof(MissionListingServer.addMisison))]
+        internal static bool SkipNullMissions(ActiveMission m) => m != null;
+
+        [HarmonyILManipulator]
+        [HarmonyPatch(typeof(SAAddMissionToHubServer), nameof(SAAddMissionToHubServer.Trigger))]
+        internal static void NullCheckOnAssignmentTag(ILContext il)
+        {
+            // Adds a local (index 8) of type 'string' to the method
+            il.Body.Variables.Add(new VariableDefinition(
+                il.Module.ImportReference(typeof(string))));
```

Comment:
Instead of making a new local, I believe you should be able Dup the value and Pop in the case where it is null.